docs/prepaid/response-code.md

```diff
@@ -2,39 +2,40 @@
 
 Below is all of response code for prepaid transaction.
 
-Response code | Description | Status
----------|----------|---------
- 00 | SUCCESS | Success
- 06 | TRANSACTION NOT FOUND | Failed
- 07 | FAILED | Failed
- 10 | REACH TOP UP LIMIT USING SAME DESTINATION NUMBER IN 1 DAY | Failed
- 12 | BALANCE MAXIMUM LIMIT EXCEEDED | Failed
- 13 | CUSTOMER NUMBER BLOCKED | Failed
- 14 |	INCORRECT DESTINATION NUMBER | Failed
- 16 | NUMBER NOT MATCH WITH OPERATOR | Failed
- 17 | INSUFFICIENT DEPOSIT | Failed
- 20 | CODE NOT FOUND | Failed
- 39 | PROCESS | Pending
- 102 | INVALID IP ADDRESS | Failed
- 106 | PRODUCT IS TEMPORARILY OUT OF SERVICE | Failed
- 107 | ERROR IN XML FORMAT | Failed
- 117 | PAGE NOT FOUND | Failed
- 121 | MONTHLY TOP UP LIMIT EXCEEDED | Failed
- 131 | TOP UP REGION BLOCKED FOR PLAYER | Failed
- 141 | INVALID USER ID / ZONE ID / SERVER ID / ROLENAME | Failed
- 142 | INVALID USER ID | Failed
- 201 | UNDEFINED RESPONSE CODE | Pending
- 202 | MAXIMUM 1 NUMBER 1 TIME IN 1 DAY | Failed
- 203 | NUMBER IS TOO LONG | Failed
- 204 | WRONG AUTHENTICATION | Failed
- 205 | WRONG COMMAND | Failed
- 206 | THIS DESTINATION NUMBER HAS BEEN BLOCKED | Failed
- 207 | 	MAXIMUM 1 NUMBER WITH ANY CODE 1 TIME IN 1 DAY | Failed
+Response code | Description | Status | Solution
+---------|----------|---------|---------
+ 00 | SUCCESS | Success | Transaction success.
+ 06 | TRANSACTION NOT FOUND | Failed | There is no transaction with your inputted ref_id. Please check again your inputted ref_id to find your transaction.
+ 07 | FAILED | Failed | Your current transaction has failed. Please try again.
+ 10 | REACH TOP UP LIMIT USING SAME DESTINATION NUMBER IN 1 DAY | Failed | Your current destination number top up request is reaching the limit on that day. Please try again tomorrow.
+ 12 | BALANCE MAXIMUM LIMIT EXCEEDED | Failed | -
+ 13 | CUSTOMER NUMBER BLOCKED | Failed | Your custom number (customer_id) has been blocked by Indobest Artha Kreasi. You can change your customer number (customer_id) or ask IAK’s Customer Service.
```

Comment:
Your customer number (customer_id) has been blocked. You can change your customer number (customer_id) or contact our Customer Service.